src/compiler/transformers/add-static-style.ts

```diff
@@ -31,18 +31,23 @@ export const addStaticStyleGetterWithinClass = (
 
 /**
  * Adds static "style" property to the class variable.
+ *
  * ```typescript
  * const MyComponent = class {}
  * MyComponent.style = "styles";
  * ```
+ *
  * @param styleStatements a list of statements containing style assignments to a class
  * @param cmp the metadata associated with the component being evaluated
  */
 export const addStaticStylePropertyToClass = (styleStatements: ts.Statement[], cmp: d.ComponentCompilerMeta): void => {
   const styleLiteral = getStyleLiteral(cmp, false);
   if (styleLiteral) {
-    const statement = ts.createStatement(
-      ts.createAssignment(ts.createPropertyAccess(ts.createIdentifier(cmp.componentClassName), 'style'), styleLiteral)
+    const statement = ts.factory.createExpressionStatement(
```

Comment:
Just wanna verify that `createExpressionStatement` is the equivalent of `createStatement`?

Reply:
yes, one way you can check is if you can find a `.createStatement` call somewhere in the codebase there should be a type hint with a deprecation message which gives the equivalent `ts.factory` function